gas/ChangeLog

```diff
@@ -1,3 +1,6 @@
+2022-07-29  Charlie Keaney  <charlie.keaney@embecosm.com>
+	* gas/config/tc-riscv.c: Fixed bug that allowed use of invalid operand 64 with cm.jt.
```

Comment:
Missing blank line, same point about `ChangeLog.COREV`

Reply:
Fixed in next patch.